gitlab-mixin/dashboards/gitlab-overview.libsonnet

```diff
@@ -0,0 +1,725 @@
+local grafana = import 'github.com/grafana/grafonnet-lib/grafonnet/grafana.libsonnet';
+local dashboard = grafana.dashboard;
+local template = grafana.template;
+local prometheus = grafana.prometheus;
+
+local dashboardUid = 'gitlab-overview';
+local matcher = 'job=~"$job", instance=~"$instance"';
+local datasourceRegex = '(?!grafanacloud-usage|grafanacloud-ml-metrics).+';
```

Comment:
This is grafana cloud specific, and shouldn't be in the OSS mixin.

Reply:
removed 👍